scripts/library.sh

```diff
@@ -787,18 +787,18 @@ function shellcheck_new_files() {
 function latest_version() {
```

Comment:
The scripts are vendored from github.com/knative/hack and copied to `scripts/` for backward compatibility, so you need to update the scripts in the hack repo first.